stream-chat-android-ui-uitests/src/androidTest/java/io/getstream/chat/android/uitests/snapshot/compose/messages/MessageItemTest.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.uitests.snapshot.compose.messages
+
+import io.getstream.chat.android.compose.state.messages.list.MessageItemState
+import io.getstream.chat.android.compose.ui.messages.list.MessageItem
+import io.getstream.chat.android.uitests.snapshot.compose.ComposeScreenshotTest
+import io.getstream.chat.android.uitests.util.TestData
+import org.junit.Test
+
+class MessageItemTest : ComposeScreenshotTest() {
+
+    @Test
+    fun messageItemForTheirsMessage() = runScreenshotTest {
```

Comment:
I see that you use Theirs insted of Their, which i think is correct. Other than looks good.